bgpd/bgp_routemap.c

```diff
@@ -2410,51 +2412,110 @@ route_set_aspath_replace(void *rule, const struct prefix *dummy, void *object)
 	as_t configured_asn;
 	char *buf;
 	char src_asn[ASN_STRING_MAX_SIZE];
+	char *acl_list_name = NULL;
+	uint32_t acl_list_name_len = 0;
+	char *buf_acl_name = NULL;
+	static const char asp_acl[] = "as-path-access-list";
+	struct as_list *aspath_acl = NULL;
 
 	if (path->peer->sort != BGP_PEER_EBGP) {
-		zlog_warn(
-			"`set as-path replace` is supported only for EBGP peers");
-		return RMAP_NOOP;
+		zlog_warn("`set as-path replace` is supported only for EBGP peers");
+		goto end_ko;
```

Comment:
What does `ko` mean?

Reply:
KO vs OK

Reply:
ko = fail

Reply:
I think this change is useless. please revert those two lines